### PR TITLE
Adding a glossary

### PR DESCRIPTION
**Pull request by christian-herber-nxp**

### CODE REVIEW

File: docs/source/x_ext.rst [hunk 1]
```diff
@@ -746,9 +746,9 @@ Memory (request/response) interface
   The memory request interface can be used by the |coprocessor| to initiate data side memory read or memory write transactions. All memory transactions, no matter if
   they are initiated by |processor| itself or by a |coprocessor| via the memory request interface, are treated equally. Specifically this equal treatment applies to:
 
-  * PMA checks and attribution
-  * PMU usage
```
Comment (christian-herber-nxp) [inline docs/source/x_ext.rst:750]: @Silabs-ArjanB : I came across the PMU term when doing the glossary. I don't know what this is, but assumed PMP is meant from the context. Is this correct?

**Review by jquevremont — APPROVED** [review]
LGTM

File: docs/source/x_ext.rst (hunk 1 above)
Comment (Silabs-ArjanB) [reply docs/source/x_ext.rst:750]: You could replace that item by two items: PMP, MMU

Comment (christian-herber-nxp) [reply docs/source/x_ext.rst:750]: MMU is actually the next item in the list. So I assume it's ok and we can merge this with PMU replace by PMP.

**Pull request merged**